docs/adguard-for-windows/browser-assistant.md

```diff
@@ -21,7 +21,7 @@ Background information:
 
 In previous versions of the AdGuard Browser Extension (before v3.5), if you had it installed along with the AdGuard for Windows app, the extension took over the functions of Assistant, stopped filtering itself and relied on the app to do this.
 
-The extension and the app worked in conjunction but sometimes the integration would fail. Some websites would stop being protected by the desktop program and be backed up only by the less powerful extension.
+The extension and the app worked in conjunction but sometimes the integration would fail. The protection normally provided by the desktop program would stop on certain sites, where only the less powerful extension would then provide protectio.
```

Comment:
protectio — это как заклинание из гарри поттера
```suggestion
The extension and the app worked in conjunction but sometimes the integration would fail. The protection normally provided by the desktop program would stop on certain sites, where only the less powerful extension would then provide protection.
```

Reply:
хаха